server/fakedb/Schedule.csv

```diff
@@ -0,0 +1,22 @@
+﻿eventId,start,end,locationLatitude,locationLongitude,repeat
```

Comment:
startTime, endTime. Just start and end can be too abstract and become a confusion later on